.circleci/config.yml

```diff
@@ -105,6 +105,25 @@ jobs:
           https://github.com/go-redis/redis
           contrib/go-redis/redis/vendor/github.com/go-redis/redis
 
+    - run:
+        name: Checkout k8s.io/client-go
```

Comment:
Are we not able to vendor only `k8s.io/client-go` instead of having both of these steps which are related? Basically following this comment, which is the suggested solution: https://github.com/kubernetes/client-go/issues/741#issuecomment-580957445

We should likely call this step something like `Fix k8s.io/client-go breaking change`

Reply:
We are not actually vendoring `k8s.io/client-go`. 

This shouldn't be a `git` checkout, but a `go get`. We just need to `go get k8s.io/client-go` before we vendor `gnostic`, because if we put a vendor directory in `$GOPATH/src/k8s.io/client-go` before we `go get k8s.io/client-go`, then `go get` will fail.

We are unable to use the suggested solution, because it requires the use of go modules.

Reply:
I see. Thanks for clearing that out. As I was looking at the issue I had less knowledge of it than you.